aqua/qt5/Portfile

```diff
@@ -19,6 +19,12 @@ version             ${qt_version}
 set middle_name     everywhere-opensource
 set worksrcdir_middle_name everywhere
 
+# qt5-qtbase is broken on macOS 15+, so fix needs to be done at the top level as all subports depends on it.
+# because of `error: 'CGDisplayCreateImageForRect' is unavailable: obsoleted in macOS 15.0 - Please use ScreenCaptureKit instead.`
+platform 24 {
```

Comment:
Should be `platform darwin 24 {`?

Reply:
No:
```tcl
platform darwin 24 {
```
is used in #25749, and that fails as well with `port` v2.10.99 (i.e, it evaluates to `false` on M1 running macOS 15.0 with Xcode 16.0 and updated command-line tools).

Reply:
> Should be `platform darwin 24 {`?

Yes, you can't use just a version with `platform`. The `24` is going to be compared with `${os.arch}` and `${os.platform}` and match neither.

Reply:
Yeah - I messed this up in one of my many changes.

Reply:
Fixed in 226b685a6f387db2b315f2b1e2022d4c778eb54a